build/amis/packer/packer.json

```diff
@@ -3,7 +3,7 @@
     "aws_access_key": "",
     "aws_secret_key": "",
     "build_timestamp": "{{timestamp}}",
-    "kubernetes_version": "1.13.2-00",
+    "kubernetes_version": "1.13.3-00",
```

Comment:
Same here. It may be worth testing if the packages we want to install are there.

Reply:
I don't think a Goss test is needed here because the Ansible playbook will fail if the specified version of the package is not installed.